selfdrive/car/honda/carcontroller.py

```diff
@@ -127,7 +127,7 @@ def update(self, CC, CS, now_nanos):
     actuators = CC.actuators
     hud_control = CC.hudControl
     conversion = hondacan.get_cruise_speed_conversion(self.CP.carFingerprint, CS.is_metric)
-    hud_v_cruise = hud_control.setSpeed / conversion if hud_control.speedVisible else 255
+    hud_v_cruise = hud_control.setSpeed / conversion if hud_control.speedVisible and not CC.cruiseControl.override else 255
```

Comment:
Do you know if all Honda platforms hide set speed on gas press? What about all Honda Bosch?

Reply:
Not so sure

Reply:
I think the behavior you’re describing also occurs on my nidec Honda. 

There are some exceptions though (the speed shows up again temporarily if you adjust the set speed while overriding).

Reply:
Interesting, so the behavior occurs across all Hondas then. I did see that exception, I don’t care much for it, unless you think it’s important?

Reply:
Well, it will probably make the code more complicated which is disappointing.  (unless the cluster implements this logic for us? I assume it doesn’t because you’re asking about it)

 But, with the set speed hidden it it’s hard to know what you’ve adjusted the set speed to. You can still see it in the openpilot ui, but I find myself looking at the cluster most of the time. 

If you ask me, I would trade the extra code complexity for the ability to see the set speed adjustments.

Reply:
@rjsmith1999 can you send a route with your car blinking the set speed?